Sources/APNSwift/LiveActivity/APNSClient+LiveActivity.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  APNSClient+LiveActivity.swift
+//  PushSender
+//
+//  Created by csms on 20/09/2022.
+//
+
+import NIOCore
+import Logging
+
+extension APNSClient {
+    /// Sends an alert notification to APNs.
+    ///
+    /// - Parameters:
+    ///   - notification: The notification to send.
+    ///
+    ///   - deviceToken: The hexadecimal bytes that identify the user’s device. Your app receives the bytes for this device token
+    ///    when registering for remote notifications.
```

Comment:
This is not true according to the Apple docs
```suggestion
    ///   - deviceToken: The hexadecimal bytes that identify the user’s device. Your apps receives the bytes after you started the live activity.
    ///    when registering for remote notifications.
```